src/illuminatio/illuminatio.py

```diff
@@ -55,7 +60,27 @@ def cli(incluster, kubeconfig):
             exit(1)
 
 
+@cli.command(short_help="generate test cases")
+@click.option(
+    "-o",
+    "--outfile",
+    default=STD_IDENTIFIER,
+    help="Output file to write results to. Format is chosen according to file ending. Supported: YAML, JSON",
+)
+def generate(outfile: str):
+    """
+    "Generate and output test cases.
+    """
+    generator = NetworkTestCaseGenerator(LOGGER)
+    v1net = k8s.client.NetworkingV1Api()
+    orch = NetworkTestOrchestrator([], LOGGER)
+    net_pols = v1net.list_network_policy_for_all_namespaces()
+    cases = generator.generate_test_cases(net_pols.items, orch.current_namespaces)[0]
```

Comment:
I would prefer `cases, _ = generator.generate_test_cases(net_pols.items, orch.current_namespaces)` to indicate that we don't use the second return value.